mysql_schema_configuration/variables.tf

```diff
@@ -20,6 +20,12 @@ variable "optional_parameters" {
   default     = {}
 }
 
+variable "optional_ssm_parameter_tags" {
+  type = "map"
```

Comment:
Is it because its a map that you cannot put it with the others optional_parameters ?

Reply:
exactly